pal/src/misc/dbgmsg.cpp

```diff
@@ -834,7 +834,7 @@ bool DBG_ShouldCheckStackAlignment()
 #include "CoreFoundation/CFString.h"
 #include "Security/AuthSession.h"
 
-static const char * PAL_DISPLAY_DIALOG PAL_GLOBAL = "PAL_DisplayDialog";
+#define PAL_DISPLAY_DIALOG "PAL_DisplayDialog"
```

Comment:
Clang OSX 7.3.0 doesn't support custom priority for non member variables and no reason that particular member is being initialized with a special priority